presto-docs/src/main/sphinx/release/release-0.257.rst

```diff
@@ -0,0 +1,27 @@
+=============
+Release 0.257
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix query exceeding local memory limit when unspilling data due to incorrect memory size calculation.
+* Fix deadlock for queries with join and limit when spilling is enabled.
+* Fix expressions with multiple ``IS NULL`` on the same variable.
+* Improve memory usage of queries spilling in the join operator.
+* Improve plan expressions to be closer to valid SQL.
+* Add support for instance count in function :func:`array_position`.
+* Add coercion support when correlated symbols are in non-direct column reference type.
```

Comment:
This is more confusing to me, and also in the end this PR added support for all kinds of expressions, not just casts.  Maybe say

`Add support for correlated subqueries with complex expressions in the correlation`

Reply:
done